syntax = "proto3";
package auth;

service AuthService {
  rpc VerifyToken(VerifyRequest) returns (VerifyResponse);
}

message VerifyRequest {
  string token = 1;
}

message VerifyResponse {
  bool loggedIn = 1;
}

// python -m grpc_tools.protoc --proto_path=protos --python_out=./pb2 --grpc_python_out=./pb2 protos/auth.proto